frontend/src/components/c-title.vue

```diff
@@ -0,0 +1,39 @@
+<template lang="pug">
+.title(v-html="markdownText", v-if="markdownText != ''")
+</template>
+
+<script lang="ts">
+import MarkdownIt from 'markdown-it';
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  data() {
+    return {
+      markdownText: '',
+    };
+  },
+  beforeMount() {
+    try {
+      fetch('title.md').then((response) => {
+        if (!response.ok) { // file not found
+          return '';
+        }
+        return response.text();
+      }).then((text) => {
+        const md = new MarkdownIt();
+        this.markdownText = md.render(text);
+      });
+    } catch (error) {
+      this.markdownText = (error as Error).toString();
+    }
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+.title {
+  padding: 2rem 1.5rem;
```

Comment:
Suggestion: I would make the vertical padding zero so that the title takes as little vertical space as possible. Moreover, vertical spacing can easily be formatted with Markdown as well.
```suggestion
  padding: 0 1.5rem;
```